core/services/product_attributes.rb

```diff
@@ -191,47 +239,47 @@ module ProductAttributes
         files_location: 'cookbooks/mariadb/files',
         reverse_product: 'mariadb_remove'
     },
-    'mariadb_plugin_columnstore' => {
+    'mariadb_ci_plugin_columnstore' => {
```

Comment:
The name of the plugin has been changed. Did you discuss the change with Timofey?